### DISCUSSION TITLE
Mail not getting sent

### DISCUSSION THREAD

**Discussion by arghya139** [postalserver/postal#2053] [Help with using Postal]
I configured this on a VPS with Ubuntu, everything said ok but mails not being sent 
![53dbbb19f8638d33d1fce2f4f0d097aa](https://user-images.githubusercontent.com/33219231/177098322-dfc30275-cc3d-44d1-bd7d-6fcfdc3e93bb.png)

**Comment by willpower232** [comment]
You should confirm that all parts of Postal are running with `postal status`.

**Reply by arghya139** [reply]
![a4c5ea641e854543a179b13dd7bb1ace](https://user-images.githubusercontent.com/33219231/177106886-1fc54dd3-cb4d-4b2c-868e-998c0f6453a4.png)

Two services are not running, how do I get them running ?

**Reply by willpower232** [reply]
I think just running `postal start` should do the trick. I'd imagine that your RabbitMQ was not ready yet when the server started.

**Reply by arghya139** [reply]
I am getting same error 

![2fa084ccc6b2f173b9e99fd55918921e](https://user-images.githubusercontent.com/33219231/177107783-e29a9065-625b-49ba-8328-831c29b128f5.png)

**Reply by willpower232** [reply]
`postal logs worker` should tell you what the issue is, given you can log in, I think the problem will be RabbitMQ

**Reply by lenny1150** [reply]
I have the same issue however, once i restart - it will come back up - but then it will stop sending again , i would say in a few hours???

why ?

**Reply by willpower232** [reply]
You'll have to look at the logs to see what the problem was
